ui/raidboss/data/06-ew/trial/golbez-ex.txt

```diff
@@ -1,6 +1,394 @@
+1057.0 "Gale Sphere 2" sync / 1[56]:[^:]*:Gale Sphere:845[89AB]:/
+1057.0 "Arctic Assault" sync / 1[56]:[^:]*:Golbez:8461:/
+1060.4 "Gale Sphere 3" sync / 1[56]:[^:]*:Gale Sphere:845[89AB]:/
+1063.8 "Gale Sphere 4" sync / 1[56]:[^:]*:Gale Sphere:845[89AB]:/
+1065.0 "Void Aero III/Tornado" sync / 1[56]:[^:]*:Golbez:845[CD]:/
+1069.1 "Phases of the Blade (cast)" sync / 1[56]:[^:]*:Golbez:86DB:/
+1071.0 "Phases of the Blade (front)" sync / 1[56]:[^:]*:Golbez:86DD:/
+1072.7 "Phases of the Blade (back)" sync / 1[56]:[^:]*:Golbez:86F2:/
+1082.1 "Binding Cold" sync / 1[56]:[^:]*:Golbez:84B3:/
+1098.2 "Azdaja's Shadow" sync / 1[56]:[^:]*:Golbez:8478:/ jump 1200.0
+1098.2 "--sync--" sync / 1[56]:[^:]*:Golbez:8479:/ jump 1400.0
+
+# repeated for timeline rolling
+
+1103.4 "Flames of Eventide"
+1106.5 "Flames of Eventide"
+1109.6 "Flames of Eventide"
+1116.7 "Void Stardust (preview)"
+1122.8 "Void Stardust"
+
+# Mini-phase 3, exaflares -> pairs -> role stacks or light parties -> out then light parties
+
+1200.0 "Azdaja's Shadow" sync / 1[56]:[^:]*:Golbez:8478:/
+1205.2 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+1208.3 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+1211.4 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+1213.4 "--jump--" sync / 1[56]:[^:]*:Golbez:84B8:/
+1218.5 "Void Stardust (preview)" sync / 1[56]:[^:]*:Golbez:84A4:/
+1224.6 "Void Stardust" duration 2.3 sync / 1[56]:[^:]*:Golbez:84A6:/
+1230.4 "Abyssal Quasar" sync / 1[56]:[^:]*:Golbez:84AB:/
+1235.4 "Eventide Fall?" sync / 1[56]:[^:]*:Golbez:8485:/
+1240.4 "Eventide Triad?" sync / 1[56]:[^:]*:Golbez:8480:/
+
+1251.4 "Phases of the Shadow (cast)" sync / 1[56]:[^:]*:Golbez:86E7:/ window 15,15
+1253.2 "Rising Beacon (record)" sync / 1[56]:[^:]*:Shadow Dragon:848D:/
+1253.2 "Phases of the Shadow (front)" sync / 1[56]:[^:]*:Golbez:86E9:/
+1254.9 "Phases of the Shadow (back)" sync / 1[56]:[^:]*:Golbez:86EF:/
+1256.1 "Rising Beacon (out)" sync / 1[56]:[^:]*:Golbez:86EC:/
+1258.2 "--sync--" sync / 1[56]:[^:]*:Golbez:8475:/
+1258.6 "Immolating Shade (light parties)" sync / 1[56]:[^:]*:Golbez:8496:/ jump 1600.0
+
+# timeline rolling
+
+1284.1 "Dragon's Descent"
+1286.5 "Explosion"
+1287.2 "Double Meteor"
+1288.8 "Cauterize"
+1310.5 "Azdaja's Shadow"
+
+# Mini-phase 3, exaflares -> pairs -> role stacks or light parties -> in then spread
+
+1400.0 "Azdaja's Shadow" sync / 1[56]:[^:]*:Golbez:8479:/
+1405.2 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+1408.3 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+1411.4 "Flames of Eventide" sync / 1[56]:[^:]*:Golbez:847B:/
+1413.4 "--jump--" sync / 1[56]:[^:]*:Golbez:84B8:/
+1418.5 "Void Stardust (preview)" sync / 1[56]:[^:]*:Golbez:84A4:/
+1424.6 "Void Stardust" duration 2.3 sync / 1[56]:[^:]*:Golbez:84A6:/
+1430.4 "Abyssal Quasar" sync / 1[56]:[^:]*:Golbez:84AB:/
+1435.4 "Eventide Fall?" sync / 1[56]:[^:]*:Golbez:8485:/
+1440.4 "Eventide Triad?" sync / 1[56]:[^:]*:Golbez:8480:/
```

Comment:
If the mechanic is `Fall`, it happens 5s faster than if it's `Triad`, I think to allow players longer to get into role stack positions.

Reply:
And yes, you always get one `Fall` and one `Triad`, the order is random but one will happen at ~535 and one will happen at ~1440 (original non-branched timeline positions are ~260 and ~441).

I have at least one pull that had Triad earlier, that one wiped before getting to the second iteration though.

```
255.2 "Void Stardust" #sync / 1[56]:[^:]*:Golbez:84AA:/
258.3 "Abyssal Quasar" sync / 1[56]:[^:]*:Golbez:84AB:/
263.2 "Eventide Triad" sync / 1[56]:[^:]*:Golbez:8480:/
```

Reply:
```
433.7 "Abyssal Quasar" sync / 1[56]:[^:]*:Golbez:84AB:/
434.6 "Lingering Spark" sync / 1[56]:[^:]*:Golbez:8468:/
438.7 "Lingering Spark" sync / 1[56]:[^:]*:Golbez:846A:/
443.7 "Eventide Fall" sync / 1[56]:[^:]*:Golbez:8485:/
```

Yeah, it's not a huge deal, mostly curious because I had this in one of my timelines, where `Fall` was at the `Triad` time.

Reply:
I think the latest push addresses these issues.